src/DeterministicVI.jl

```diff
@@ -16,16 +16,23 @@ export ElboArgs
 
 """
 ElboArgs stores the arguments needed to evaluate the variational objective
-function
+function.
 """
 type ElboArgs{NumType <: Number}
     S::Int64
     N::Int64
+
+    # The number of components in the point spread function.
+    psf_K::Int64
```

Comment:
In fact, this is preparing for representing the PSF with a convolution, which requires a PSF_free image of the sky that can be convolved with a pixelated PSF.  Furthermore, in the PSF-free image, a star cannot be rendered as a single point, since then any ```u``` derivative will be zero (the image as a function of the location of the star will be discontinuous).  So I'm rendering the PSF-free image with a single PSF component that is about a half-pixel wide and truncating the BVN evaluations at a small number of standard deviations.

This is all entirely separate from inferring the PSF from the image.  I will do that in a future CL with a different data structure for shared parameters.